examples/dev-novel/main.ts

```diff
@@ -496,6 +496,7 @@ storiesOf("SearchBox")
     wrapWithHits({
       template: `
         <ais-search-box
+          [setFocus]="true"
```

Comment:
is this normal Angular convention? I'd expect this prop to be called `focused` or `autofocus` (like the DOM prop)

Reply:
If I read this right, [instantsearch.js is using autofocus](https://github.com/algolia/instantsearch.js/blob/9632ade6434fb9257bdb23c66dde8aaa1c31c4eb/src/widgets/search-box/search-box.js#L17). Maybe let use autofocus for the sake of consistency?

Reply:
sure will change it accordingly.